### PR TITLE
[NewUI] Lint fixes.

### PR DESCRIPTION
**Pull request by danjm**
Ran `npm run lint` and fixed lint errors.

### CODE REVIEW

File: ui/app/components/dropdowns/account-options-dropdown.js [hunk 1]
```diff
@@ -14,12 +14,13 @@ module.exports = AccountOptionsDropdown
...
 AccountOptionsDropdown.prototype.render = function () {
-  const { selected, network, identities, style, dropdownWrapperStyle, menuItemStyles } = this.props
+  // const { selected } = this.props
+  const { network, identities, style, dropdownWrapperStyle, menuItemStyles } = this.props
 
   return h(AccountDropdowns, {
     enableAccountOptions: true,
     enableAccountsSelector: false,
-    selected: selectedAddress,
+    // selected: selectedAddress,
```
Comment (chikeichan) [inline ui/app/components/dropdowns/account-options-dropdown.js]: What is this change? It seems to be more than linting fixes

File: app/scripts/chromereload.js [hunk 2]
```diff
@@ -32,7 +32,7 @@
...
 
-(function e (t, n, r) { function s (o, u) { if (!n[o]) { if (!t[o]) { var a = typeof require === 'function' && require; if (!u && a) return a(o, !0); if (i) return i(o, !0); var f = new Error("Cannot find module '" + o + "'"); throw f.code = 'MODULE_NOT_FOUND', f } var l = n[o] = {exports: {}}; t[o][0].call(l.exports, function (e) { var n = t[o][1][e]; return s(n ? n : e) }, l, l.exports, e, t, n, r) } return n[o].exports } var i = typeof require === 'function' && require; for (var o = 0; o < r.length; o++)s(r[o]); return s })({1: [function (require, module, exports) {
+(function e (t, n, r) { function s (o, u) { if (!n[o]) { if (!t[o]) { var a = typeof require === 'function' && require; if (!u && a) return a(o, !0); if (i) return i(o, !0); var f = new Error("Cannot find module '" + o + "'"); throw f.code = 'MODULE_NOT_FOUND', f } var l = n[o] = {exports: {}}; t[o][0].call(l.exports, function (e) { var n = t[o][1][e]; return s(n || e) }, l, l.exports, e, t, n, r) } return n[o].exports } var i = typeof require === 'function' && require; for (var o = 0; o < r.length; o++)s(r[o]); return s })({1: [function (require, module, exports) {
```
Comment (chikeichan) [inline app/scripts/chromereload.js]: Let's ignore app/scripts/*

File: ui/app/components/dropdowns/account-selection-dropdown.js [hunk 3]
```diff
@@ -14,12 +14,12 @@ module.exports = AccountSelectionDropdown
...
 AccountSelectionDropdown.prototype.render = function () {
-  const { selected, network, identities, style, dropdownWrapperStyle, menuItemStyles } = this.props
+  // const { selected } = this.props
+  const { network, identities, style, dropdownWrapperStyle, menuItemStyles } = this.props
 
   return h(AccountDropdowns, {
     enableAccountOptions: false,
     enableAccountsSelector: true,
-    selected: selectedAddress,
     network,
```
Comment (chikeichan) [inline ui/app/components/dropdowns/account-selection-dropdown.js]: Same here. Why is `selected` removed?

File: ui/app/components/pending-tx.js [hunk 4]
```diff
@@ -74,18 +72,24 @@ PendingTx.prototype.getTotal = function () {
   const { params = [] } = decodedData || {}
   const { name, value } = params[1] || {}
   const amountBn = name === '_value'
-    ? new BN(value)
-    : hexToBn(txParams.value)
+    ? value
+    : txParams.value
```
Comment (chikeichan) [inline ui/app/components/pending-tx.js]: Seems to be unrelated to linting

**Review by chikeichan — COMMENTED** [review]
The PR seems to have commits unrelated to lint fixes. Can you either remove the non-lint fixes commit, or create a new PR that only contain the lint fixes?

File: ui/app/components/dropdowns/account-options-dropdown.js (hunk 1 above)
Comment (danjm) [reply ui/app/components/dropdowns/account-options-dropdown.js]: `selectedAddress` is not defined. So this fixes the no-undef lint error without changing the values of `selected` property as passed to in `AccountDropdowns`

File: ui/app/components/dropdowns/account-selection-dropdown.js (hunk 3 above)
Comment (danjm) [reply ui/app/components/dropdowns/account-selection-dropdown.js]: Same as above. `selectedAddress` is not defined.

File: ui/app/components/pending-tx.js (hunk 4 above)
Comment (danjm) [reply ui/app/components/pending-tx.js]: Yes, unrelated to linting. This branch had commits on it that it should not have... due to the reverting of some branches that was done last week. I have removed those commits.

**Comment by chikeichan** [comment]
@danjm can you look into resolving the conflicts?

**Comment by danjm** [comment]
The changes here have gone stale. I'll open another PR with a similar purpose later.
